relayer/listener.go

```diff
@@ -209,6 +208,7 @@ func (lstnr *Listener) processLogs(ctx context.Context) error {
 			// variables such as Quorum values and processing missed blocks.
```

Comment:
I think we can get rid of this TODO. If implemented, we'd likely still want to mark the relayer as unhealthy after some period of time or number of failed retries. To me, there's no functional difference between that and simply retrying a fixed number of times before marking as unhealthy, as we do now.

Reply:
I removed the TODO. Thinking about this more I think we should not fatal on the unhealthy status and let the outside caller of the binary decide what to do and how long to give it to become healthy again.